src/libs/antares/study/parameters.h

```diff
@@ -433,11 +432,13 @@ class Parameters final
         PowerFluctuations fluctuations;
     } power;
 
-    struct
+    struct UCMode
     {
         //! Unit Commitment Mode
         UnitCommitmentMode ucMode;
-    } unitCommitment;
+        void addExcludedVariables(std::vector<std::string>&) const;
```

Comment:
Not obvious: we could add a comment why this is needed

Reply:
done